src/pages/Profile.js

```diff
@@ -0,0 +1,12 @@
+import Navigation from "./components/Navigation";
+
+export default function Profile() {
+  return (
+    <section>
+      <div>create profile page</div>
+      <footer className="Footer">
+        <Navigation />
+      </footer>
+    </section>
+  );
```

Comment:
- maybe give the section a classname?
- and i would use instead of <div> maybe some h-tag.